main/sigmabot.py

```diff
@@ -192,7 +174,12 @@ async def bots(ctx):
 ## MINING DIFFICULTY ADJUSTMENTS
 @client.command()
 async def difficulty(ctx):
-        await ctx.send(fq.faq_bots())
+        await ctx.send(fq.faq_difficulty())
+
+## SEED
+@client.command()
+async def difficulty(ctx):
```

Comment:
@cafebedouin for future reference: you'll also want to change this as well otherwise the bot command for seed will need to call `.difficulty` which is a conflict with the one above